symplyphysics/laws/kinematic/rotational_inertia_about_axis_and_through_com.py

```diff
@@ -0,0 +1,58 @@
+from sympy import Eq
+from symplyphysics import (
+    units,
+    Symbol,
+    Quantity,
+    validate_input,
+    validate_output,
+    print_expression,
+)
+
+# Description
+## The parallel-axis theorem relates the rotational inertia of a body about any axis to
+## that of the same body about a parallel axis that extends through the body's com (center
+## of mass).
+
+# Law: I = I_com + M*(h**2)
+## I - rotational inertia about some axis
+## I_com - rotational inertia about such an axis that is parallel to the given one
+##         and that passes through the com (center of mass)
+## M - body mass
+## h - perpendicular distance between the two axes
+
+# Conditions:
+## - The two axis must be parallel to each other.
+## - One of them should pass through the body's com.
```

Comment:
I think not just one of them should pass, but the one that we've used for `I_com`